test/controllers/change_requests_controller_test.rb

```diff
@@ -12,11 +12,6 @@ class ChangeRequestsControllerTest < ActionController::TestCase
     assert_not_nil assigns(:change_requests)
   end
 
-  test 'should get new' do
-    get :new
-    assert_response :success
-  end
```

Comment:
The `change_requests` routes explicitly exclude the `new`, `edit`, and `update` actions.